_notifications-plugin/index.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: default
+title: Notifications
+nav_order: 1
+has_children: false
+redirect_from:
+  - /notifications-plugin/
+---
+
+# Notifications
+
+The Notifications plugin provides a central location for all of your notifications from OpenSearch plugins. Using the plugin, you can configure which communication service you want to use and see relevant statistics and troubleshooting information. Currently, the Alerting and ISM plugins have integrated with the Notifications plugin.
+
+You can use either OpenSearch Dashboards or the REST API to configure notifications. Dashboards offers a more organized way of selecting a channel type and selecting which OpenSearch plugin sources you want to use, whereas the REST API lets you programmatically define your notification channels for better versioning and reuse later on.
+
+1. Use the Dashboards UI to first create a channel that receives notifications from other plugins. Supported communication channels include Amazon Chime, Amazon Simple Notification Service (Amazon SNS), Amazon Simple Email Service (Amazon SES), email through SMTP, Slack, and custom webhooks when selecting how you want the plugin to send notifications. After you’ve configured your channel and plugin sources, send messages and start tracking your notifications from the Notifications plugin's dashboard.
```

Comment:
Do you mean on line 14? Starting with that might be a bit misleading since users can't control how the Notification plugin sends notifications on the backend.

Reply:
I was referring to this sentence: Supported communication channels include Amazon Chime, Amazon Simple Notification Service (Amazon SNS), Amazon Simple Email Service (Amazon SES), email through SMTP, Slack, and custom webhooks when selecting how you want the plugin to send notifications.

Reply:
Actually, I'll just take out "when selecting how you want the plugin to send notifications." I don't super like how it's there now that I read it again.